packages/remix-node/upload/fileUploadHandler.ts

```diff
@@ -228,9 +228,7 @@ export class NodeOnDiskFile implements File {
     return readableStreamToString(this.stream());
   }
 
-  public get [Symbol.toStringTag]() {
-    return "File";
-  }
+  public readonly [Symbol.toStringTag] = "File";
```

Comment:
https://typescript-eslint.io/rules/class-literal-property-style: enforces a consistent style for declaring class literals. The default is to prefer a `public readonly` over a `public get` because it's less code. 

Note that the rule provides [_suggestions_ instead _fixes_](https://eslint.org/docs/latest/extend/custom-rules#providing-suggestions) (i.e. its suggestions can only be done manually, not by `--fix`) because they technically change code behavior. But this seems to be equivalent to me, and I couldn't find any issues?